pkg/controller/operators/olm/operator.go

```diff
@@ -1444,7 +1454,25 @@ func (a *Operator) syncCopyCSV(obj interface{}) (syncError error) {
 		return err
 	}
 
+	// Ensure that the Copied CSVs exist in the protected namespaces.
+	protectedNamespaces := []string{}
+	for ns := range a.protectedCopiedCSVNamespaces {
+		if ns == clusterServiceVersion.GetNamespace() {
+			continue
+		}
+		protectedNamespaces = append(protectedNamespaces, ns)
+	}
+
+	if err := a.ensureCSVsInNamespaces(clusterServiceVersion, operatorGroup, NewNamespaceSet(protectedNamespaces)); err != nil {
+		logger.WithError(err).Info("couldn't copy CSV to protected Copied CSV namespaces")
+		syncError = err
```

Comment:
Should we add a comment for why we're falling through here vs. returning early?

Reply:
This should have been returned as an error, addressing.